docker-compose.yml

```diff
@@ -92,13 +91,13 @@ services:
       start_period: 40s
 
 volumes:
-  bux-postgresql-data:
+  wallet-postgresql-data:
     driver: local
-  bux-redis-data:
+  wallet-redis-data:
     driver: local
-  bux-mongodb-data:
+  wallet-mongodb-data:
     driver: local
-  pulse-data:
+  block-headers-service-data:
     driver: local
 
 configs:
```

Comment:
we should use localhost:8180 as apiUrl

also we should probably also add `wsUrl` property which I forgot about.

in other words, json should look like that:
```json
      {
          "paymailDomain": "${RUN_PAYMAIL_DOMAIN}",
          "apiUrl": "http://localhost:8081",
          "wsUrl": "ws://localhost:8081/api/websocket"
      }
```

Reply:
added